lib/mongoid/contextual/memory.rb

```diff
@@ -188,15 +188,16 @@ def limit(value)
         self
       end
 
+      # Pluck the field values in memory.
+      #
+      # @example Get the values in memory.
+      #   context.pluck(:name)
+      #
+      # @param [ Array<String, Symbol> ] *fields Field(s) to pluck.
```

Comment:
We do not use the `@param ... *parameter` syntax currently and it looks confusing. Is each element of `fields` an array or is the entire `fields` an array?

Can you please reference where this syntax is defined, and if it doesn't have an established definition, either remove `*` or remove `Array<>` leaving `[ String | Symbol ]`.

Reply:
Good point, I've gone with:

```
@param [ String | Symbol ] *fields Field(s) to pluck.
```

Since `*fields` is what appears in the method signature `def pluck(*fields)`. This is unambiguous that it's a splat. `Array` doesn't feel right because Arrays aren't actually passed in. I think we should adopt this convention for splats everywhere (I'm happy to raise a follow-up ticket to do so.)

Reply:
One more comment, I think we should clarify in other methods that `|` is used for the union of types, while `,` should be reserved for `Hash<Symbol, String>` i.e. separation of args within a generic type.